roles/openstack-stack/defaults/main.yml

```diff
@@ -18,4 +18,4 @@ dns_volume_size: 1
 lb_volume_size: 5
 use_bastion: False
 ui_ssh_tunnel: False
-provider_network: None
+provider_network: False
```

Comment:
Why is this necessary? Jinja's conditionals treat `None` as falsey. What is this breaking for CASL?

Reply:
@tomassedovic I didn't capture the error, but running this with Ansible v2.3.1.x, the error was something like "provider_network 'None' not valid".

Reply:
i.e.: error coming from the Ansible modules, not CASL specific per-say

Reply:
Hm, I'm running `2.3.1.0` and had never seen it. Might be a Jinja version or something. Whatever, I'll merge it.